### DISCUSSION TITLE
Cutlass transpose matrix multiplication

### DISCUSSION THREAD

**Discussion by YoniSp** [NVIDIA/cutlass#548] [Q&A]
I I am trying to compute A*Bt by defining B matrix as transposed matrix during gemm configuration without any success.

What is the appropriate way to compute A*Bt succesfully?

**Comment by hwu36** [comment]
You just need to specify the layout (row major or column major) of A, B, C, D

**Reply by YoniSp** [reply]
Lets assume that  A and B are  column major matrices.
 The way to multiply A* Bt is changing B layout to row major  instead of column major?

**Comment by hwu36** [comment]
I am not sure i understand your question correctly or not.  But i think the answer is yes.
